docs/manual/logging.rst

```diff
@@ -0,0 +1,30 @@
+=======
+Logging
+=======
+
+Sentry
+======
+
+To monitor and identify issues for OpenZaak, it is recommended to use `Sentry`_.
+If Sentry is configured for OpenZaak, all exceptions that occur within the application
+are sent to Sentry. For documentation on how to set up a project in Sentry, please refer to the official `documentation`_.
+After setting up the project, you will receive a **DSN**. The created Sentry project can be linked to OpenZaak by setting
+the environment variable ``SENTRY_DSN`` equal to this DSN.
+
+.. _`sentry`: https://sentry.io/
+.. _`documentation`: https://docs.sentry.io/error-reporting/quickstart/?platform=javascript
```

Comment:
I think https://docs.sentry.io/guides/getting-started/ is a better starting point. You'll probably want to point out that for the project they should pick `Django` as platform. It's also useful to give an example `DSN` so they know what to copy-paste from the code examples.